.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -100,12 +100,20 @@ body:
 
         ##### Release on MavenCentral
 
+        - [ ] Checkout the branch `main`
         - [ ] Run the command `./gradlew publish --no-daemon --no-parallel`. You'll need some non-public element to do so
+        - [ ] Run the command `./gradlew closeAndReleaseRepository`. If it is working well, you can jump directly to the final step of this section.
```

Comment:
:raised_hands: much simpler!

Reply:
I should also take some time to write the github action to release